File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/talk/TalkData.java
package jp.co.terraresta.androidlesson.data.model.talk;

import com.google.gson.annotations.Expose;

import java.util.List;

import jp.co.terraresta.androidlesson.data.model.common.BaseResultData;

/**
 * Created by ooyama on 2017/05/26.
 */

public class TalkData extends BaseResultData {

    @Expose
    public List<TalkItem> items = null;

    public List<TalkItem> getItems() {
        return items;
    }
}

File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/talk/TalkData.kt
package jp.co.terraresta.androidlesson.data.model.talk

import com.google.gson.annotations.Expose

import jp.co.terraresta.androidlesson.data.model.common.BaseResultData

/**
 * Created by ooyama on 2017/05/26.
 */

class TalkData : BaseResultData() {

    @Expose
    var items: List<TalkItem>? = null
}
